src/webgpu/api/validation/createRenderPipeline.spec.ts

```diff
@@ -737,3 +737,7 @@ g.test('shader_module,device_mismatch')
 
     t.doCreateRenderPipelineTest(isAsync, _success, descriptor);
   });
+
+g.test('entry_point_name_must_match')
+  .desc('TODO: Do similarly to createComputePipeline but for both vertex and fragment.')
```

Comment:
I'm confused because this file is `createRenderPipeline.spec.ts`. Perhaps you mean to say createRenderpipeline